android/sdl_android/src/main/java/com/smartdevicelink/managers/video/VideoStreamManager.java

```diff
@@ -205,7 +205,7 @@ private synchronized void checkState(){
 		if(this.getState() == SETTING_UP
 				&& isTransportAvailable
 				&& hmiLevel != null
-				&& hmiLevel.equals(HMILevel.HMI_FULL)
```

Comment:
Do `isStreaming()` and is `isPaused()` need to be updated as well? Please double-check how iOS handle that. Also, all usages for HMILevel in this class should be revisited to make sure they make sense for streaming video in `LIMITED`.

Reply:
Agreed! Added!